Logger/LogFactory.cs

```diff
@@ -1,11 +1,44 @@
-﻿namespace Logger
+﻿using System;
+namespace Logger
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+
+        private string? _FilePath;
+        private string? FilePath
         {
-            
-            return null;
+            get => _FilePath!;
+
+            set
+            {
+                if (value == null) { throw new ArgumentNullException(nameof(value)); }
+
+                _FilePath = value;
+            }
+        }
+
+
+        public BaseLogger? CreateLogger(string className, string? filePath)
```

Comment:
```suggestion
        public BaseLogger? CreateLogger(string className)
```

You should be using the filePath set from the `ConfigureFileLogger` method, not passing in a new filePath